docs/user-guide/layer-settings.md

```diff
@@ -134,6 +139,110 @@ The editor is easy to approach thanks also to the following functions:
 !!!warning
     The *autocomplete* and the *color picker* functions are available only in the CSS editor.
 
+### Visual Editor Style
+
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) also allows to edit the layers style using a *Visual editor* with a most user friendly UI.Clicking on the <img src="../img/button/visual_editor_style_button.jpg" class="ms-docbutton"/> button a section opens so that the user can customize the style through with a visual stile editor by adding editing symbolizers, which can be: *Mark*, *Icon*, *Line*, *Fill* and *Text*. It is anyway possible to switch to the text editor mode if necessary for a more complex styling.
+
+<img src="../img/layer-settings/visual_editor_style.jpg" class="ms-docimage"  style="max-width:500px;">
+
+Once a symbolizer has been added and customized, you can:
+
+<img src="../img/layer-settings/style_options.jpg" class="ms-docimage"  style="max-width:500px;">
+
+* **Filter** he style rule, as explained [here](filtering-layers.md#attribute-filter), by clicking the <img src="../img/button/filter_white_button.jpg" class="ms-docbutton"/> button.
+
+* Add a **Scale denominator filter** (`max` and `min` scale) to visualize the style rule only within certain scale limits. This is possible by clicking the <img src="../img/button/scale_denominator_button.jpg" class="ms-docbutton"/> button.
+
+* **Remove** the symbolizer by clicking the <img src="../img/button/delete_white_button.jpg" class="ms-docbutton"/> button.
+
+#### Mark
+
+The mark type allows you to add a mark to the layer: clicking on the <img src="../img/button/add_mark_button.jpg" class="ms-docbutton"/> button a mark panel appears: 
+
+<img src="../img/layer-settings/mark_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+The mark can have different `Shape`, `Color`, `Stroke` with different `Color` and `Width` and customizable `Radius` and `Rotation`. Take a look at the following example.
+
+<img src="../img/layer-settings/mark_style_ex.jpg" class="ms-docimage">
+
+#### Icon 
+
+With the icon panel, which opens by clicking on <img src="../img/button/add_icon_button.jpg" class="ms-docbutton"/> button, the style editor is allowed to add an image as an icon (by specifying its *URL*) and customize the icon `Opacity`, `Size` and `Rotation` angle:
+
+<img src="../img/layer-settings/icon_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+#### Line
+
+The line rule is used to style linear features of the layer: clicking on the <img src="../img/button/add_line_button.jpg" class="ms-docbutton"/>  button a panel allows the user to edit the corresponding properties. 
+
+<img src="../img/layer-settings/line_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+The editor can change a `Stroke color`, a `Stroke width`, a `Line style` (*continuous*, *dashed*, etc), the `Line cap` (*Butt*, *Round*, *Square*) and the `Line join` (*Bevel*, *Round*, *Miter*). An example can be the following one:
```

Comment:
The editor can change the `Stroke color`, the `Stroke width`, the `Line style` (*continuous*, *dashed*, etc), the `Line cap` (*Butt*, *Round*, *Square*) and the `Line join` (*Bevel*, *Round*, *Miter*). An example can be the following one: